### PR TITLE
chore: set max SDK version to <3.0.0

### PR DESCRIPTION
**Pull request by chalin**
cc @kwalrath @kevmoo

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.dart_tool/
 .packages
 .pub/
```
Comment (kevmoo) [inline .gitignore]: Feel free to remove `.pub` and `build/`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
+## 0.1.2
```
Comment (kevmoo) [inline CHANGELOG.md]: Make it 0.1.1+2

File: .travis.yml [hunk 3]
```diff
@@ -9,17 +9,20 @@ dart:
...
+  - dartfmt
+  - dartanalyzer
+  - test: --platform vm
+    xvfb: false
+  # Uncomment this line to run tests on the browser.
+  - test: --platform firefox
 
 # Only run one instance of the formatter and the analyzer, rather than running
 # them against each Dart version.
 matrix:
```
Comment (kevmoo) [inline .travis.yml]: There is no `-stable`, so no need to exclude them, too. Just delete this section.

File: .travis.yml [hunk 4]
```diff
@@ -20,10 +20,13 @@ dart_task:
...
   exclude:
-  - dart: stable
-    dart_task: dartfmt
-  - dart: stable
-    dart_task: dartanalyzer
+    - dart: stable
+      dart_task: dartfmt
+    - dart: stable
+      dart_task: dartanalyzer
+  allow_failures:
```
Comment (kevmoo) [inline .travis.yml]: We need to get these tests passing!

File: .gitignore (hunk 1 above)
Comment (chalin) [reply .gitignore]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (chalin) [reply CHANGELOG.md]: Done

File: .travis.yml (hunk 3 above)
Comment (chalin) [reply .travis.yml]: It is acting as a placeholder for when we resume building across two channels. Do you still want me to remove these lines?

File: .travis.yml (hunk 4 above)
Comment (chalin) [reply .travis.yml]: Do you want me to give fixing these a try, or continue down the list of packages first?

File: .travis.yml [hunk 5]
```diff
@@ -9,18 +9,22 @@ dart:
...
 
 # Only run one instance of the formatter and the analyzer, rather than running
 # them against each Dart version.
 matrix:
-  include:
-  - dart: dev
-    dart_task: dartfmt
-  - dart: dev
-    dart_task: dartanalyzer
+  exclude:
```
Comment (kevmoo) [inline .travis.yml]: There is no stable, just remove the matrix

Comment (chalin) [reply .travis.yml]: Done

**Pull request merged**